examples/vector_fe/vector_fe_ex9/run.sh

```diff
@@ -4,7 +4,7 @@
 
 source $LIBMESH_DIR/examples/run_common.sh
 
-example_name=vector_fe_ex8
+example_name=vector_fe_ex9
```

Comment:
Can't believe we missed this before.  I guess it worked because we never use this argument for a directory name, just for header/footer output?

Reply:
yea I believe that's the case